### DISCUSSION TITLE
Academy: "Administration of Pimcore & Data Importer" Having doubt to get the correct answers

### DISCUSSION THREAD

**Discussion by adi84900** [pimcore/pimcore#15284] [Q&A / Support] [answered]
**_What can be defined via permissions out of the box in Pimcore Admin Interface?_**

![image](https://github.com/pimcore/pimcore/assets/32416123/882b9023-5b6b-4e0f-881e-83d29d388415)

Unable to get the correct answers for the above question

**Comment by adi84900** [comment]
@fashxp @willers-pm @NiklasBr @Rajesh17692
 Could you please help here

**Comment by willers-pm** [comment, marked as answer]
If I recall correctly, these are the items to be checked:
- Permissions to most of main menu entries.
- Visible Custom Layouts and Perspectives.
- Access rules for Pimcore elements via workspaces.
No guarantees, though, feel free to let me know if these are right.

**Reply by adi84900** [reply]
Thanks @willers-pm
